src/react-components/link-root.js

```diff
@@ -106,7 +111,8 @@ class LinkRoot extends Component {
               <div className={styles.enteredDigits}>
                 <input
                   className={styles.digitInput}
-                  type="number"
+                  type="tel"
+                  pattern="[0-9]*"
```

Comment:
ah interesting, i didn't realize this was considered a bug. it seemed logical that a "0001" entry for a `number` seems wrong. i'll add a comment. not sure about internationalisation. even previously it was just a rendering issue, if they just punch the value in the keypad as you'd expect i think it will work no matter what.